dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpChannelConfig.java

```diff
@@ -2166,13 +2151,12 @@ private void parseHeaderChangeLimit(Map<Object, Object> props) {
      *
      * @param props
      */
-    private void parseRequestSmugglingProtection(Map<Object, Object> props) {
+    protected void parseRequestSmugglingProtection(Object option) {
         // PK53193 - allow this to be disabled
-        Object value = props.get(HttpConfigConstants.PROPNAME_ENABLE_SMUGGLING_PROTECTION);
-        if (null != value) {
-            this.bEnableSmugglingProtection = convertBoolean(value);
-            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
-                Tr.debug(tc, "Config: request smuggling protection is " + this.bEnableSmugglingProtection);
+        if (Objects.nonNull(option)) {
+            this.bEnableSmugglingProtection = convertBoolean(option);
+            if (TraceComponent.isAnyTracingEnabled() && tc.isEventEnabled()) {
+                Tr.event(tc, "Config: request smuggling protection is " + this.bEnableSmugglingProtection);
```

Comment:
This was changed from a debug to an event, I'm assuming for consistency but should we do this separately?